lib/selenium/screenshot.js

```diff
@@ -106,8 +106,10 @@ function tryAndLoadUrl(url, driver, deferred) {
  * @param {object} task
  * @param {object} cachedDrivers
  */
-function getDriver(task, cachedDrivers) {
-    var browserShortName = task.browser;
+function getDriver(task, cachedDrivers, sizeName) {
+    var isMobile = sizeName && sizeName === 'xs',
+        browserShortName = isMobile ? `${task.browser}-mobile` : task.browser,
```

Comment:
Note: When running the CLI I get an error because the lookup fails:

```
[02:41 PM moollaza@use-devtesting4 ddg-screen-diff {bhall/mobile-user-agent}]$ ./lib/run.js search "apples" -s xs
taking 1 screenshot in 1 batch
taking screenshot for http://use-devtesting4.duckduckgo.com/?q=apples on headless-chromium at 420x755 (xs)
page possibly didn't load - taking a screenshot anyway
error! TypeError: Cannot read property 'quit' of undefined
    at /home/moollaza/projects/ddg-screen-diff/lib/selenium/screenshot.js:233:49
    at ManagedPromise.invokeCallback_ (/home/moollaza/projects/ddg-screen-diff/node_modules/selenium-webdriver/lib/promise.js:1379:14)
    at TaskQueue.execute_ (/home/moollaza/projects/ddg-screen-diff/node_modules/selenium-webdriver/lib/promise.js:2913:14)
    at TaskQueue.executeNext_ (/home/moollaza/projects/ddg-screen-diff/node_modules/selenium-webdriver/lib/promise.js:2896:21)
    at /home/moollaza/projects/ddg-screen-diff/node_modules/selenium-webdriver/lib/promise.js:2775:27
    at /home/moollaza/projects/ddg-screen-diff/node_modules/selenium-webdriver/lib/promise.js:639:7
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    ```

Reply:
Pushed a commit to fix this